CONTRIBUTING.md

```diff
@@ -95,8 +95,17 @@ In multi-line bullet point entries, subsequent lines should be indented.
 - Insert a blank line before at-clauses/tags.
 - Favor `{@code foo}` over `<code>foo</code>`.
 - Favor literals (e.g., `{@literal @}`) over HTML entities.
+- New classes and methods should have `@since ...` annotation  
 - Use `@since 5.0` instead of `@since 5.0.0`.
-- Do not use `@author` tags. Instead, contributors are listed on [GitHub](https://github.com/junit-team/junit5/graphs/contributors).
+- Do not use `@author` tags. Instead, contributors are listed on [GitHub](https://github.com/junit-team/junit5/graphs/contributors). 
+- Do not use verbs in third person form (e.g. use "Discover tests..." instead of "Discover tests..." 
+  in the first sentence describing a method). 
+
+#### JavaDoc Examples
+
+See [here](java\org\junit\jupiter\api\extension\ExtensionContext.java) and 
+[here](java\org\junit\jupiter\api\extension\ParameterContext.java) for example javadoc
```

Comment:
These links don't work, do they?

Reply:
I wanted to link to the files using the relative paths in the repository. 

Apologies for that.